lib/cmorizer.rb

```diff
@@ -15,10 +15,30 @@ def initialize(src_txt)
       @experiments = []
       @years_step = 1
       @eval_mode = true
+      src_txt = sanitize src_txt
       instance_eval(src_txt, src_txt)
       @eval_mode = false
     end
     
+    def sanitize(src_txt)
```

Comment:
I think is in general a better practice to have a more descriptive name of the function, for example `handle_uppercase_vars`. In this way when you are reading the code in line 18 you already have an idea of what this function is doing without having to look at the function itself

Reply:
Plus 1 to this comment 😄 (But take with a grain of salt: I am a little bit neurotic about naming things)

Reply:
Sure, that make sense to be more precise about what this function does by giving it a proper name.  This was not a priority thing than get the logic right at that time but thanks for reading through and suggesting a proper name.

I still need to modify the logic a little bit in deciding where to insert the bug fix line. In the current implementation it is introduced just a line above where it find a Capitalised variable line but I want to this to move to the top of the file in case some other function depends on the same issue being fixed.